### DISCUSSION TITLE
Generating a figure like Figure 6 of your published paper

### DISCUSSION THREAD

**Discussion by msreza007** [jotech/gapseq#167] [Q&A]
Dear gapseq developers

Thanks for developing such a nice tool. I have used Gap-seq for pathway analysis of one bacteria. I am now wondering how can I generate a metabolism model like Figure 6 which will include production and consumption data for top 10 metabolites of my analysed bacteria. 

Could you please share the R code you used to generate figure 6? 

Thanks in advance. 

Salman

**Comment by Waschina** [comment]
Do you refer to Figure 6 or 4 in the gapseq paper? https://doi.org/10.1186/s13059-021-02295-1

**Reply by msreza007** [reply]
My apologies. I said Fig 6 from Biorxiv but in fact its figure 4 of your publication. I will be appreciative if you can share the code

**Comment by Waschina** [comment]
The code is available via this github-Repository: https://github.com/Waschina/gapseqEval

Directory: AnaerobicFoodWeb
